### ISSUE TITLE
Install guide not correct

### ISSUE THREAD

**Issue by vincent-pli** [google/ko#337]
In README it said:
```
VERSION=TODO # choose the latest version
OS=Linux     # or Darwin
ARCH=x86_64  # or arm64, i386, s390x
curl -L https://github.com/google/ko/releases/download/${VERSION}/ko_${VERSION}_${OS}_${ARCH}.tar.gz | tar xzf - ko
chmod +x ./ko
```

But it not work, for example, the `VERSION=v0.8.1`, but the tar name is `ko_0.8.1_Linux_x86_64.tar.gz` rather than `ko_v0.8.1_Linux_x86_64.tar.gz`

**Comment by ImJasonH** [comment]
This was going to be fixed by #325 

I'll fix it now since that was blocked by CLA hurdles.

Thanks for pinging this.

**Comment by ImJasonH** [comment]
Fixed https://github.com/google/ko/commit/0621cbac8e0384de9307b3698ef091aedf751b2b
